package-lock.json

```diff
@@ -2337,6 +2337,7 @@
         "@woocommerce/navigation": "2.1.0",
         "@wordpress/components": "7.4.0",
         "@wordpress/compose": "3.3.0",
+        "@wordpress/date": "3.3.0",
```

Comment:
I noticed this getting added in another branch I did too.  I'm guessing it may be resulting from a renovate merge at some point.

Reply:
I'm not sure why this appeared all of a sudden because `@woocommerce/components` had `@wordpress/date` as a dependency from long time ago:
https://github.com/woocommerce/woocommerce-admin/blob/master/packages/components/package.json#L31
Any idea? In any case, I don't think we should modify `package-lock.json` by hand, right?

Reply:
I'm okay with leaving it.